src/index.coffee

```diff
@@ -47,6 +47,21 @@ pipe = flip compose
 
 spread = (f) -> (ax) -> f ax...
 
+wait = (f) -> (x) -> if x?.then? then (x.then (a) -> f a) else (f x)
+
+flow = (fx...) ->
+  if fx?.length == 1
+    if fx[0]? && Object.getPrototypeOf(fx[0]) == Array.prototype
+      flow fx[0]...
+  else if fx?
+    (ax...) ->
+      [start, fx...] = fx
```

Comment:
Ah … I just realized that there's another difference between `pipe` and `flow`: the latter takes an initial argument. So we're still missing straight async composition.